<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOwnerDialog</class>
 <widget class="QDialog" name="AddOwnerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>addOwnerDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="warningBalanceDoubleSpinBox">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="warningBalanceLabel">
       <property name="text">
        <string>Warning balance:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="commentsTextEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="commentsLabel">
       <property name="text">
        <string>Comments:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="hideCheckBox">
       <property name="text">
        <string>Hide</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddOwnerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddOwnerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
